cmd/snap/cmd_snap_op.go

```diff
@@ -872,6 +893,79 @@ func (x *cmdRefresh) Execute([]string) error {
 	return x.refreshMany(names, opts)
 }
 
+func (x *cmdRefresh) holdRefreshes() (err error) {
+	var opts client.SnapOptions
+
+	if x.Hold == "forever" {
+		opts.Time = "forever"
+	} else {
+		dur, err := time.ParseDuration(x.Hold)
+		if err != nil {
+			return fmt.Errorf("hold value must be a number of hours or minutes (e.g., 72h): %v", err)
+		}
+
+		opts.Time = timeNow().Add(dur).Format(time.RFC3339)
+	}
+
+	names := installedSnapNames(x.Positional.Snaps)
+	var changeID string
+	if len(names) == 1 {
+		changeID, err = x.client.HoldRefreshes(names[0], &opts)
+	} else {
+		changeID, err = x.client.HoldRefreshesMany(names, &opts)
+	}
+
+	if err != nil {
+		return err
+	}
+
+	_, err = x.wait(changeID)
+	if err != nil {
+		if err == noWait {
+			return nil
+		}
+		return err
+	}
+
+	if len(names) == 0 {
+		fmt.Fprintf(Stdout, i18n.G("Held auto-refreshes of all snaps until %s\n"), opts.Time)
+	} else {
+		fmt.Fprintf(Stdout, i18n.G("Held general refreshes of %s until %s\n"), strutil.Quoted(names), opts.Time)
+	}
+
+	return nil
+}
+
+func (x *cmdRefresh) unholdRefreshes() (err error) {
+	names := installedSnapNames(x.Positional.Snaps)
+	var changeID string
+	if len(names) == 1 {
+		changeID, err = x.client.UnholdRefreshes(names[0], nil)
+	} else {
+		changeID, err = x.client.UnholdRefreshesMany(names, nil)
+	}
+
+	if err != nil {
+		return err
+	}
+
+	_, err = x.wait(changeID)
+	if err != nil {
+		if err == noWait {
+			return nil
+		}
+		return err
+	}
+
+	if len(names) == 0 {
+		fmt.Fprintf(Stdout, i18n.G("Removed hold on auto-refreshes of all snaps\n"))
```

Comment:
```suggestion
		fmt.Fprintf(Stdout, i18n.G("Removed auto-refresh hold on all snaps\n"))
```

Reply:
Similar to above, just to remain consistent and to try and avoid other _hold_ connotations.

Reply:
done